src/components/layout/layout.module.scss

```diff
@@ -6,31 +6,42 @@
   position: relative;
 }
 
-.container {
+.content {
+  display: flex;
+
+  // Make sure the texture is behind the content
   position: relative;
   z-index: 1;
-  max-width: 1020px;
+}
+
+.main {
+  max-width: min(calc(100vw - 170px), 1024px);
```

Comment:
The `max-width` attribute is calculated imperatively to avoid the content overflowing the `flex` container, which results in a horizontal scrollbar and "white column" with overflown content.

(Picture taken from https://api3.eth.link/#/history/secondary-5, should not happen anymore)
![prod](https://user-images.githubusercontent.com/22679154/129479073-2776a7d0-ac47-438c-9358-c97279427dad.png)